tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/IApplicationBuilder.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="IApplicationBuilder.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+#if !NETFRAMEWORK
+using System.Collections.Generic;
+using Datadog.Trace.DuckTyping;
+using Microsoft.AspNetCore.Http;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore
+{
+    /// <summary>
+    /// appbuilder
+    /// </summary>
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public interface IApplicationBuilder
```

Comment:
might want to enable `#nullable`, not definitely worth it

Reply:
fixed in [ef58dfb](https://github.com/DataDog/dd-trace-dotnet/pull/3442/commits/ef58dfbb0ad8fc4f3297dac3eca7eb91655b72ea)